contributor-guide/modules/ROOT/pages/docs/documentation-guidelines.adoc

```diff
@@ -1,15 +1,37 @@
 = Guidelines
 :navtitle: Guidelines
 
-It is not a requirement for a Boost library for the documentation to adhere to the following structure. However, it is listed here as a guide, if needed.
+This section covers how to structure library documentation and where to locate it. There should be an updated version of the documentation for each release, even if there are few or no changes to the actual text.
+
+== Location
+
+The target location for completed and published library documentation should match the following:
+
+[source,asciidoc]
+----
+https://www.boost.org/doc/libs/<VERSION>/libs/<LIBRARY NAME>/doc/html/index.html
+----
+
+For example:
+
+[source,asciidoc]
+----
+https://www.boost.org/doc/libs/1_81_0/libs/json/doc/html/index.html
+https://www.boost.org/doc/libs/1_82_0/libs/serialization/doc/index.html
+https://www.boost.org/doc/libs/1_83_0/libs/beast/doc/html/index.html
+----
+
+== Structure
+
+It is not a _requirement_ for a Boost library for the documentation to adhere to the following structure. However, it is listed here as a guide, if needed.
```

Comment:
Actually, there _is_ a structure that's required. That's the index.html entry point as generated by the `boost:<lib-name>[]` macro in asciidoc. Usually, this entry point can already have some content or redirect the user anywhere else, like `doc/html/index.html`. Have a look at the script and the links it generates.